fuzzers/044-clk-bufg-pips/bits.dbf

```diff
@@ -0,0 +1,32 @@
+26_07 26_08 27_06,CLK_BUFG.CLK_BUFG_BUFGCTRL0_I0.CLK_BUFG_IMUX28_0
```

Comment:
Would be good to add some information about how this file was worked out / created? Maybe that will be in your README?

Reply:
None of the other fuzzers have ever needed that?